testsuite/pom.xml

```diff
@@ -52,7 +52,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
I already commented about the error given running the integration tests by this line. Integration tests pass now, so the PR looks ok to me.